packages/bot-web-ui/src/components/dashboard/dashboard.scss

```diff
@@ -201,6 +207,17 @@
         &--expanded {
             width: calc(100vw - 39.8rem);
         }
+
+        &--modal {
+            // position: fixed !important;
+            z-index: 99999999 !important;
+            width: 400px !important;
+            height: 500px !important;
+            // top: 0;
```

Comment:
why comments are not removed ?

Reply:
missed it, thanks for bringing it up, I've fixed✅